components/LinkGrid/src/index.scss

```diff
@@ -166,6 +192,16 @@
     display: none;
   }
 
+  .denhaag-link-grid__container {
+    padding-inline-end: 0;
+  }
+
+  .denhaag-link-grid__grid {
+    @media (min-width: 1280px) and (max-width: 1767px) {
+      grid-template-columns: repeat(4, 1fr) !important;
```

Comment:
Je kunt ook de `grid-template-columns` in een variabel stoppen zoals:
**_in de tokens de waarde "repeat(2, 1fr)"_**, daarna overschijven in de media queries, dan hoef je geen !importants te gebruiken.
```

.denhaag-link-grid__grid {
  @media (min-width: 768px) {
    grid-template-columns: var(--denhaag-link-grid-grid-template-columns);
  }

  @media (min-width: 1280px) {
   --denhaag-link-grid-grid-template-columns: repeat(3, 1fr);
  }

 @media (min-width: 1280px) and (max-width: 1767px) {
   --denhaag-link-grid-grid-template-columns: repeat(4, 1fr);
  }
}
```